manifests/w/WarcraftLogs/Uploader/5.9.5.162/WarcraftLogs.Uploader.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: WarcraftLogs.Uploader
+PackageVersion: 5.9.5.162

```

```suggestion
PackageVersion: 5.9.5
```
Should match control panel